build.spec

```diff
@@ -50,9 +51,16 @@ hiddenimports: list[str] = [
 ]
 
 if sys.platform == "linux":
-    # Needed files for better system tray support on Linux via pystray (AppIndicator backend).
-    datas.append((Path("/usr/lib/girepository-1.0/AppIndicator3-0.1.typelib"), "gi_typelibs"))
-    binaries.append((Path("/lib/x86_64-linux-gnu/libappindicator3.so.1"), "."))
+    arch = platform.machine()
+    if arch == "x86_64":
```

Comment:
I'm thinking this whole section can be simplified to something like this (untested):

```py
datas.append((Path(f"/usr/lib/{arch}-linux-gnu/girepository-1.0/AyatanaAppIndicator3-0.1.typelib"), "gi_typelibs"))
binaries.append((Path(f"/usr/lib/{arch}-linux-gnu/libayatana-appindicator3.so.1"), "."))
```

Because there's no real reason to artificially limit this to `x86_64` and `aarch64`, given that Ubuntu supports other architectures as well...

Reply:
yeah that's a good idea i guess, the only issue i might see happening is some weird os not using that naming scheme, but i guess that's kinda a niche case compared to disallowing it, i'll pr this with the other review